.github/workflows/workflow.yml

```diff
@@ -22,7 +22,9 @@ jobs:
       fail-fast: false
     steps:
       - name: Git checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
+        with:
+          depth: 0
```

Comment:
I believe this is named `fetch-depth`: https://github.com/actions/checkout/blob/c952173edf28a2bd22e1a4926590c1ac39630461/action.yml#L56

Reply:
Fixed